src/mono/mono/mini/interp/interp.c

```diff
@@ -1761,6 +1761,18 @@ ves_pinvoke_method (
 	interp_pop_lmf (&ext);
 
 #ifdef MONO_ARCH_HAVE_INTERP_PINVOKE_TRAMP
+#ifdef MONO_ARCH_HAVE_SWIFTCALL
+	if (mono_method_signature_has_ext_callconv (sig, MONO_EXT_CALLCONV_SWIFTCALL)) {
+		int arg_index;
+		gpointer data = mono_arch_get_swift_error (&ccontext, sig, call_info, &arg_index);
+
+		// Perform an indirect store at arg_index stack location
+		if (data != NULL) {
```

Comment:
Why would `data` ever be null here ? Shouldn't we assert instead ?

Reply:
If SwiftError arg is not passed, the data will be NULL. The code is updated to first check if the arg is found and then asserts.